.github/workflows/ci-deepspeech-v2.yml

```
@@ -0,0 +1,32 @@
+name: Continuous Integration for Deepspeech Version 2
```

Adjust it to the other workflows:
```suggestion
name: Continuous Integration Deepspeech v2
```